maps/graph_edges/glitched_logic/dgb_jumpless_mega_rush.py

```diff
@@ -0,0 +1,9 @@
+"""
+This file represents edges of the world graph that have to be added
+for Glitched Logic: Jumpless Mega Rush
+Can climb the bookshelf by partially clipping into each shelf and using Parakarry.
+"""
+edges_dgb_add_jumpless_mega_rush = [
+    #* Hidden Bedroom (2F) Door Bottom -> ItemA (MegaRush)
+    {"from": {"map": "DGB_13", "id": 0}, "to": {"map": "DGB_13", "id": "ItemA"}, "reqs": [["Parakarry"]], "mapchange": False},
+]
```

Comment:
Interestingly, this *only* puts the mega rush item in logic, and not the coins on the bed. Because the coins expect boots, and there is no "bed coins w/o boots" trick yet IIRC
Could workaround this by adding an `DGB_13/ItemA > DGB_13/ItemB` edge here, in the meantime

Reply:
I honestly assumed the coins would already be in logic without jump, but I guess logic assumes you pull out the drawer so this will be quick.